docs/en/stack/ml/df-analytics/ml-dfa-classification.asciidoc

```diff
@@ -1,8 +1,303 @@
+with classes `false` and `true`, the predictions of interest are the cells where
+the actual and predicted labels are both `true` (also known as a true positive
+(TP)) or both `false` (also known as a true negative (TN)):
 
+[role="screenshot"]
+image::images/confusion-matrix-binary-accuracy.jpg[alt="A confusion matrix with the correct predictions highlighted",width="75%"]
+
+TIP: If there is an imbalanced distribution of classes in your training data 
+set, focusing on accuracy can decrease your model's sensitivity to incorrect
+predictions in the classes that are under-represented in your data.
+
+By default, {classanalysis} jobs accept a slight degradation of the overall
+accuracy in return for greater sensitivity to classes that are predicted
+incorrectly. That is to say, their objective is to maximize the minimum recall.
+For example, in the context of a multi-class confusion matrix, the predictions
+of interest are in each row:
+
+[role="screenshot"]
+image::images/confusion-matrix-multiclass-recall.jpg["A confusion matrix with a row highlighted"]
+
+For each class, the recall is calculated as the number of correct predictions
+(where the actual label matches the predicted label) divided by the sum of all
+the other predicted labels in that row. This value is represented as a
+percentage in each cell of the confusion matrix. The class scores are then
+weighted to favor predictions that result in the highest recall values across
+the training data. This objective typically performs better than accuracy when
+you have highly imbalanced data.
+
+To learn more about choosing the class assignment objective that fits your goal, 
+refer to this 
+https://github.com/elastic/examples/blob/master/Machine%20Learning/Class%20Assigment%20Objectives/classification-class-assignment-objective.ipynb[Jupyter notebook].
+
+[discrete]
+[[dfa-classification-feature-importance]]
+==== {feat-imp-cap}
+
+{feat-imp-cap} provides further information about the results of an analysis and 
+helps to interpret the results in a more subtle way. If you want to learn more 
+about {feat-imp}, <<ml-feature-importance,click here>>. 
+
+[discrete]
+[[dfa-classification-evaluation]]
+=== Measuring model performance
+
+You can measure how well the model has performed on your data set by using the 
+`classification` evaluation type of the 
+{ref}/evaluate-dfanalytics.html[evaluate {dfanalytics} API]. The metric that the 
+evaluation provides you is a confusion matrix. The more classes you have, the 
+more complex the confusion matrix is. The matrix tells you how many data points 
+that belong to a given class were classified correctly and incorrectly.
+
+If you split your data set into training and testing data, you can determine how
+well your model performs on data it has never seen before and compare the
+prediction to the actual value.
+
+[discrete]
+[[dfa-classification-readings]]
+=== Further readings
```

Comment:
> === Further readings

Unless these further readings are all specifically related to measuring model performance, I think they should be moved  out of this section (perhaps to the end of this page, to match what is done in the language identification page?)